src/components/AnonymousUsername/index.js

```diff
@@ -1,34 +1,35 @@
 import * as React from 'react';
-import {StyleSheet, Text} from 'react-native';
+import {StyleSheet, Text, TextStyle} from 'react-native';
 
 import {COLORS} from '../../utils/theme';
 import {POST_VERSION} from '../../utils/constants';
 import {fonts} from '../../utils/fonts';
 import {getOfficialAnonUsername} from '../../utils/string/StringUtils';
 
 const styles = StyleSheet.create({
-  feedUsername: {
+  feedUsername: (isFeeds) => ({
     fontFamily: fonts.inter[600],
     fontWeight: 'bold',
     fontSize: 14,
     lineHeight: 16.94,
     color: COLORS.black,
-    flex: 1
-  }
+    flex: isFeeds ? 0 : 1
+  })
 });
 
 /**
  * @typedef {Object} AnonymousUsernameComponentProps
  * @property {string} version
  * @property {AnonUserInfoTypes} anonUserInfo
+ * @property {TextStyle} style
  */
 
 /**
  *
  * @param {AnonymousUsernameComponentProps} props
  */
 const AnonymousUsername = (props) => {
-  const {version, anonUserInfo} = props;
+  const {version, anonUserInfo, isFeed} = props;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `isFeed` prop is being destructured from `props`, but it's not defined in the `AnonymousUsernameComponentProps` typedef. This could lead to potential issues with TypeScript if `isFeed` is not an optional prop. Please add `isFeed` to the `AnonymousUsernameComponentProps` typedef.

```diff
 /**
  * @typedef {Object} AnonymousUsernameComponentProps
  * @property {string} version
  * @property {AnonUserInfoTypes} anonUserInfo
+ * @property {boolean} isFeed
  * @property {TextStyle} style
  */
```

42:
The `styles.feedUsername(isFeed)` call assumes that `isFeed` is a boolean. If `isFeed` can be `undefined`, this might lead to unexpected behavior. Consider providing a default value for `isFeed`.

```diff
- <Text testID="newVersion" style={styles.feedUsername(isFeed)}>
+ <Text testID="newVersion" style={styles.feedUsername(isFeed ?? false)}>
```

48:
The `props.style` is being applied to the `Text` component but it's not clear if this prop is intended to override the existing styles or just supplement them. If it's meant to override, it should be placed after `styles.feedUsername` in the array.

```diff
- return <Text style={[styles.feedUsername, props.style]}>Anonymous</Text>;
+ return <Text style={[styles.feedUsername, ...props.style]}>Anonymous</Text>;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->